opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -563,16 +569,14 @@ def __init__(
         max_attributes: Optional[int] = None,
         max_events: Optional[int] = None,
         max_links: Optional[int] = None,
+        max_span_attributes: Optional[int] = None,
```

Comment:
I am confused by the naming, was expecting add diff for `max_attributes` based on the PR title but this has `max_span_attributes`.

Reply:
`max_attributes` already exists but really was used as `max_span_attributes`. What we sort of did here is rename `max_attributes` to `max_span_attributes` and then add `max_attributes` if that makes sense. :)